spec/features/home_page_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+describe 'Visit the home page', type: :feature do
+  before do |test|
+    # a game that has both players
+    FactoryGirl.create(:game, name: 'Full game')
+
+    if test.metadata[:open_game]
+      # a game that has only one player (player1)
+      @game2 = FactoryGirl.create(:game, name: 'First open game', player_2: nil)
+
+      # a game that has only one player (player2)
+      @game3 = FactoryGirl.create(:game, name: 'Second open game', player_1: nil)
+    end
+
+    visit root_path
+  end
```

Comment:
Later on, if we had more tests that used game2/game3. It would make sense to move them into a let.